as2-send-edi-855-hybrid-sap-order-ack/readme.md

```diff
@@ -0,0 +1,185 @@
+# Recieve 810(EDI Invoice) in B2B cloud using http/https and create an expense report in Concur using the Invoice statement.
```

Comment:
This readme is identical to the 810 one?

Reply:
I have updated all 3 examples. Can you please verify and accept